README.md

```diff
@@ -22,7 +35,133 @@ cd build/dist/bin
+- If no verbose option is set then INFO level logs are printed to the console output.   
+- You can set multiple options at the same time. Ex.: `hz start -p 9000 -J -Xmx2g -v`
+
+When you run `hz start` command with any options above, it starts a Hazelcast instance in the foreground. Please use Ctrl+C (SIGINT) to gracefully stop the running instance.
+
+### `hz mc start`
+
+This command starts a new Hazelcast Management Center instance. To display its usage info, please run `hz mc start -h`:
+
+```
+$ ./hz mc start -h
+Usage: hz mc start [-hvV] [-vv] [-c=<context-path>] [-p=<port>] [-J=<option>[,
+                   <option>...]]...
+Starts a new Hazelcast Management Center instance
+  -h, --help            Show this help message and exit.
+  -V, --version         Print version information and exit.
+  -c, --context-path=<context-path>
+                        Bind to the specified <context-path> which is the path
+                          that Management Center runs.
+  -p, --port=<port>     Bind to the specified <port>.
+  -J, --JAVA_OPTS=<option>[,<option>...]
+                        Specify additional Java <option> (Use ',' to separate
+                          multiple options).
+  -v, --verbose         Output with FINE level verbose logging.
+      -vv, --vverbose   Output with FINEST level verbose logging.
+```
+
+The following options are available with `mc start` command:
+
+- `-c, --context-path=<context-path>`: Binds the started Management Center web application to the specified context path. For instance, if you run with `hz mc start -c my-management-center` Management Center starts at `http://localhost:8080/my-management-center`. If not specified, Management Center starts at the root context (`http://localhost:8080/`). 
+- `-p, --port=<port>`: *(Default: 8080)* Binds the started Management Center web application to the specified port.
+- `-J, --JAVA_OPTS=<option>[,<option>...]`: Adds additional Java options to the JVM that is running the started Management Center. Please note that class path settings (such as `-cp`, `-jar`) are **not** allowed. Ex.: `hz mc start -J -verbose:gc,-Xms2g,-Xmx2g`
+- `-v, --verbose`: Allows to print FINE level logging to the console output.
+- `-vv, --vverbose`: Allows to print FINEST level logging to the console output. 
+
+**Notes:**
+
+- If no verbose option is set then INFO level logs are printed to the console output.   
+- You can set multiple options at the same time. Ex.: `hz mc start -p 9000 -J -Xmx2g -v`
+
+When you run `hz mc start` command with any options above, it starts a Hazelcast Management Center instance in the foreground. Please use Ctrl+C (SIGINT) to gracefully stop the running instance.
+
+### `hz -V`
+
+This command prints Hazelcast IMDG, Management Center and the tool's version information to the console output: 
+
+```
+$ ./hz -V
+CLI tool: 4.2020.08
+Hazelcast IMDG: 4.0.2
+Hazelcast Management Center: 4.2020.08
+```
 
 ## Checkstyle validation
```

Comment:
These sections should be collocated with the information on how to build the CLI from the source. And I agree that it should be at the end of the README.